Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -2693,8 +2693,10 @@ static String commandNameForSelector(SEL selector)
         m_softSpaceRange = NSMakeRange(NSNotFound, 0);
 #if HAVE(TOUCH_BAR)
     updateTouchBar();
-    if (m_page->editorState().hasPostLayoutData())
+    if (m_page->editorState().hasPostLayoutData()) {
         requestCandidatesForSelectionIfNeeded();
+        updateCaretDecorationPlacement();
```

Comment:
Seems strange that `updateCaretDecorationPlacement` is underneath `HAVE(TOUCH_BAR)`.

Reply:
Very strange indeed 😛 I shall fix